src/client/common/CMakeLists.txt

```diff
@@ -13,6 +13,8 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 add_library(client_common STATIC
+  alias_dict.cpp
+  client_formatter.cpp
```

Comment:
A little nitpick...please put `client_formatter.cpp` *after* `client_common.cpp` to be alphabetical.

Reply:
Good catch :)